piksi_tools/fileio.py

```diff
@@ -66,13 +68,14 @@ def readdir(self, dirname='.'):
     files = []
     while True:
       msg = struct.pack("<I", len(files)) + dirname + '\0'
-      self.link.send(SBP_MSG_FILEIO_READ_DIR, msg)
-      data = self.link.wait(SBP_MSG_FILEIO_READ_DIR, timeout=1.0)
+      self.link.send(SBP_MSG_FILEIO_READ_DIR_REQUEST, msg)
```

Comment:
Is `SBP_MSG_FILEIO_READ_DIR_REQUEST` in a state where we can actually construct an object and and send that over, or is it still broken w.r.t. Construct?